packages/teleport/src/services/nodes/makeNode.ts

```diff
@@ -18,11 +18,11 @@ import { at } from 'lodash';
 import { Node } from './types';
 
 export default function makeNode(json): Node {
-  const [id, clusterId, hostname, tags = [], addr, tunnel = false] = at(json, [
+  const [id, clusterId, hostname, tagTexts = json.tags.map(tag => `${tag.name}: ${tag.value}`), addr, tunnel = false] = at(json, [
```

Comment:
lets refactor this function to use the most recent way of processing responses (without lodash), similar to https://github.com/gravitational/webapps.e/blob/master/teleport/src/services/workflow/makeAccessRequest.ts#L8

Reply:
Done! :+1: